docs/source/user_guide/types.rst

```diff
@@ -215,3 +214,14 @@ You can also check the underlying PySpark data type of `Series` or schema of `Da
      |-- d: double (nullable = false)
      |-- s: string (nullable = false)
      |-- b: boolean (nullable = false)
+
+.. note::
+    Koalas currently does not support multiple types of data in single column.
+    This is because Koalas uses Spark DataFrame internally and Spark DataFrame does not support multiple types of data in single column.
```

Comment:
Use consistent wording. Looks like this page used PySpark DataFrame instead of Spark DataFrame.